src/components/VueTypeaheadBootstrapList.vue

```diff
@@ -114,13 +205,25 @@ export default {
         if (this.query.length === 0) {
           return text
         }
-        const re = new RegExp(this.escapedQuery, 'gi')
+
+        let regexp = '';
+
+        for (let letter of this.query) {
+          const normalizedLetter = normalize(sanitize(letter));
+          if (characterMap.has(normalizedLetter)) {
+            regexp += characterMap.get(normalizedLetter);
+          } else {
+            regexp += letter;
+          }
+        }
```

Comment:
If this needs to stay, can you pull this out into its own method? It seems like it deserves it's own name and a few tests demonstrating it's purpose.